### PR TITLE
ts-web/rt: playground make deposit/withdraw cross-account

### PR DESCRIPTION
**Pull request by pro-wh**
along with some fixes elsewhere to make this work

and a release of client-rt

### CODE REVIEW

File: runtime-sdk/src/modules/consensus_accounts/mod.rs [hunk 1]
```diff
@@ -215,7 +215,12 @@ impl<Accounts: modules::accounts::API, Consensus: modules::consensus::API>
 
         // Signer.
         let signer = &ctx.tx_auth_info().signer_info[0];
-        Consensus::ensure_compatible_tx_signer(ctx)?;
+        if body.to.is_none() {
+            // If no `to` field is specified, i.e. withdrawing to the transaction sender's account,
+            // only allow the consensus-compatible single-Ed25519-key signer type. Otherwise, the
+            // tokens would get stuck in an account that you can't sign for on the consensus layer.
+            Consensus::ensure_compatible_tx_signer(ctx)?;
```
Comment (kostko) [inline runtime-sdk/src/modules/consensus_accounts/mod.rs:222]: Would be nice to have a unit test that covers this case.

**Review by kostko — APPROVED** [review]

**Review by ptrus — APPROVED** [review]

**Comment by kostko** [comment]
Added tests and fixed ts-web lint.

**Comment by pro-wh** [comment]
thanks for the fixes!

File: runtime-sdk/src/modules/consensus_accounts/mod.rs (hunk 1 above)
Comment (pro-wh) [reply runtime-sdk/src/modules/consensus_accounts/mod.rs:222]: yeah. let me check if/how we currently cover this check

Comment (kostko) [reply runtime-sdk/src/modules/consensus_accounts/mod.rs:222]: I already added the tests ;-)

Comment (pro-wh) [reply runtime-sdk/src/modules/consensus_accounts/mod.rs:222]: oh great, thanks!

**Pull request merged**